politeiawww/cmsdatabase/cockroachdb/models.go

```diff
@@ -84,3 +85,20 @@ type ExchangeRate struct {
 func (ExchangeRate) TableName() string {
 	return tableNameExchangeRate
 }
+
+// Payments contains all the information about a given invoice's payment
+type Payments struct {
+	InvoiceToken    string `gorm:"not null"`
```

Comment:
The primary key isn't set for the `Payments` table so it defaults to a hidden column.

```
     column_name    | data_type | is_nullable | column_default | generation_expression |   indices   | is_hidden
+-------------------+-----------+-------------+----------------+-----------------------+-------------+-----------+
  invoice_token     | STRING    |    false    | NULL           |                       | {}          |   false
  address           | STRING    |    false    | NULL           |                       | {}          |   false
  tx_ids            | STRING    |    false    | NULL           |                       | {}          |   false
  time_started      | INT8      |    false    | NULL           |                       | {}          |   false
  time_last_updated | INT8      |    false    | NULL           |                       | {}          |   false
  amount_needed     | INT8      |    false    | NULL           |                       | {}          |   false
  amount_received   | INT8      |    false    | NULL           |                       | {}          |   false
  status            | INT       |    false    | NULL           |                       | {}          |   false
  rowid             | INT       |    false    | unique_rowid() |                       | {"primary"} |   true
```

Shouldn't the primary key be the `InvoiceToken` column?